modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/treeview/Treeview.scss

```diff
@@ -28,6 +28,10 @@ $white: #fff;
 
 			.card-body {
 				padding: 0.5rem;
+
+				&__inner {
+					min-height: 2rem;
+				}
```

Comment:
Can we use [`card-row`](https://github.com/liferay/clay/blob/master/packages/clay-card/src/Row.tsx#L19) class, instead of adding a new one?

Reply:
For sure! Thanks 👏